modin/utils.py

```diff
@@ -40,6 +40,11 @@
 PANDAS_API_URL_TEMPLATE = f"https://pandas.pydata.org/pandas-docs/version/{pandas.__version__}/reference/api/{{}}.html"
 
 MODIN_UNNAMED_SERIES_LABEL = "__reduced__"
+"""
```

Comment:
Isn't it more conventional to put this comment above the variable name?

Reply:
I agree with that being more intuitive, but convention (and documentation auto-generation tools) seem to prefer putting it below. See the example in [this PEP](https://peps.python.org/pep-0258/#attribute-docstrings), and [this example](https://www.sphinx-doc.org/en/master/usage/extensions/example_google.html) from sphinx.